composer.json

```diff
@@ -19,10 +19,11 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "simple-bus/message-bus": "~3.0",
-        "symfony/http-kernel": "~2.3 || ~3.0",
-        "symfony/dependency-injection": "~2.3 || ~3.0",
-        "symfony/yaml": "~2.3 || ~3.0",
-        "symfony/config": "~2.3 || ~3.0"
+        "symfony/http-kernel": "~2.3 || ~3.0 || ~4.0",
+        "symfony/dependency-injection": "~2.3 || ~3.0 || ~4.0",
+        "symfony/yaml": "~2.3 || ~3.0 || ~4.0",
+        "symfony/config": "~2.3 || ~3.0 || ~4.0",
+        "symfony/proxy-manager-bridge": "~2.3 || ~3.0 || ~4.0"
```

Comment:
This is a dependency for `SimpleBus/DoctrineORMBridge` right?

Reply:
Yes, but it is a Symfony specific dependency because the SimpleBus/DoctrineORMBridge can be used, from my understanding, in projects that do not use Symfony. That is why I've included it in the SymfonyBridge.